src/main/java/app/controllers/UserController.java

```diff
@@ -11,11 +11,11 @@ public class UserController {
     public static void addRoutes(Javalin app, ConnectionPool connectionPool) {
         app.post("login", ctx -> login(ctx, connectionPool));
         app.get("logout", ctx -> logout(ctx));
-        app.get("createuser", ctx -> ctx.render("createuser.html"));
-        app.post("createuser", ctx -> createUser(ctx, connectionPool));
+        app.get("createAccount", ctx -> ctx.render("createAccount.html"));
+        app.post("createAccount", ctx -> createAccount(ctx, connectionPool));
```

Comment:
Consider whether to use `createuser` or `createAccount`. If `createUser` is chosen it needs to be replaced elsewhere in the class such as in try/catch when displaying errors to the user